### ISSUE TITLE
General question: pattern names

### ISSUE THREAD

**Issue by matentzn** [obophenotype/upheno#415]
since we currently represent abnormal, should the pattern names be

abnormallyCalcifiedAnatomicalEntity? rather than calcifiedAnatomicalEntity?
abnormallyAlteredNumberOfAnatomicalEntities? rather than alteredNumberOfAnatomicalEntity?

**Comment by ybradford** [comment]
@matentzn adding 'abnormally' to the pattern name makes sense to me.

**Comment by sbello** [comment]
For the first the assumption was that the entity is not usually calcified so I think adding abnormal is fine. For number I would suggest replacing Altered with abnormal if you want consistency. I would not say abnormally altered as that implies there is some 'normal' altered number and that is not what I think we are trying to say.

**Comment by matentzn** [comment]
Yeah, I can see that, but I am trying to be consistent with the naming scheme. The pato quality is 'altered number of'. the modifier abnormal. Are you suggesting that the pato quality 'altered number of' should be replaced by 'number of' (if that exists), and then say: 'number of' and' has modifier' some abnormal? I think we can never assume, even in the case of PATO:monstrous, that the phenotypic quality is abnormal; maybe we describe the observation of some life form that is generally monstrous :P So in this sence, altered number of, all by itself, can be either normal, or abnormal!
